bioindex/lib/index.py

```diff
@@ -145,7 +145,9 @@ def build(self, config, engine, use_lambda=False, use_batch=False, workers=3, co
         Builds the index table for objects in S3.
         """
         logging.info('Finding keys in %s...', self.s3_prefix)
-        s3_objects = list(list_objects(config.s3_bucket, self.s3_prefix, only='*.json'))
+        json_objects = list(list_objects(config.s3_bucket, self.s3_prefix, only='*.json'))
+        gz_objects = list(list_objects(config.s3_bucket, self.s3_prefix, only='*.gz'))
```

Comment:
Should I make the glob *.json.gz?

Reply:
Either works. It'll quite obviously explode with anything but json.gz anyways.

Reply:
I guess `*.jzon.gz` maybe is better just for the idea of is some random log file or something is gzed into somewhere it won't interfere with the method overall.